src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -401,10 +403,17 @@ public QueryResult query(QueryRequest queryRequest) {
     }
 
     /**
-     * TODO: write Javadoc.
+     * Execute a scan meeting the specs of @{link AmazonDynamoDB}, but scoped to single tenants within a shared table.
+     * If no tenant context is specified, a multi tenant scan is performed over the multi tenant shared table.
```

Comment:
"multi tenant" -> "multitenant" (here and anywhere else; sim. with "multi-tenant" -> "multitenant")